README.md

```diff
@@ -22,8 +22,16 @@ Additional event is emitted, when user scrolls up - it allows you to refresh ele
     // your callback for reloading element, i.e.
     reloadContent($element);
   });
-
-  AdRefresher.init($element);
+  
+  // default options
+  var options = {
+    loadingThreshold: 300,
+    reloadingThreshold: 150,
+    useReloading: true        // if set to false, it will disable reload event
```

Comment:
It looks like duplicating the same information once again. The logic can be modified so if `options` does not contain `reloadingThreshold` or `reloadingThreshold` is set to 0, it means that reloading should be disabled - so you don't need `useReloading` then. What do you think?

Reply:
Currently setting thresholds to 0 will (re)load ads when the first pixel becomes visible.
Do we want to keep that behaviour?

Reply:
hmm, you're right. It is totally different behaviour. So your approach here is correct :+1: